app/views/finders/show.atom.builder

```diff
@@ -0,0 +1,21 @@
+atom_feed do |feed|
+  feed.title(@finder.name)
+
+  dates = []
+  @results.results.map(&:metadata).each do |r|
```

Comment:
Agreed that this is really heavy, but this is the only way I could figure out of getting the latest date for each document as not all documents expose `published_at` or something similar and some expose more than one date. CMA for example has Opened and Closed date. Unless anyone has a similar solution I can move this to a Presenter.